sig-scheduling/annual-report-2021.md

```diff
@@ -4,15 +4,16 @@
 
 1. What work did the SIG do this year that should be highlighted?
 
-   -
-   -
-   -
+   - Efficient re-queueing of pods, significantly cutting the number of failed scheduling cycles
+   - Improvements to preemption performance
+   - Simplified plugin configuration in component config
+   - Scheduler simulator: https://github.com/kubernetes-sigs/kube-scheduler-simulator
 
 2. What initiatives are you working on that aren't being tracked in KEPs?
 
-   -
-   -
-   -
+   - Performance improvements and benchmarking
+   - Code refactorings and cleanups
+
 
 3. KEP work in 2021 (1.x, 1.y, 1.z):
```

Comment:
any other KEPs land in 1.21 (first release in 2021) you want to call out?

Reply:
we had a couple alpha features started in 1.21 and made it to in 1.22, so they are listed under beta.